webapp/src/octoClient.ts

```diff
@@ -381,6 +381,20 @@ class OctoClient {
         }
         return (await this.getJson(response, [])) as IUser[]
     }
+
+    async getUserWorkspaces(): Promise<UserWorkspace[]> {
+        if (!Utils.isFocalboardPlugin()) {
```

Comment:
I'm not 100% sure if this is correct, the client library in theory should be usable for third party applications and looks like passing here the `isFocalboardPlugin` is an implementation detail that shouldn't belong to the library.

Reply:
Done.